src/MoBi.UI/Views/EditContainerView.cs

```diff
@@ -63,6 +66,16 @@ public override void InitializeBinding()
          btParentPath.ButtonClick += (o, e) => OnEvent(_presenter.UpdateParentPath);
       }
 
+      private void ConfirmAndExecuteContainerModeChange(ContainerMode newMode)
+      {
+         OnEvent(() =>
+         {
+            var result = _presenter.SetContainerMode(newMode);
+            if (!result)
+               cbContainerMode.SelectedIndex = 0;
+         });
+      }
```

Comment:
The method name does not match convention because it starts with capital. You could also remove reference to the confirmation because that's not done here now. If you want, you could rename the presenter method to ConfirmAndSetContainerMode since that's where the confirmation is done.

I would also provide both the oldMode and newMode in this method, then use oldMode to reverse the SelectedIndex.